support-frontend/assets/helpers/subscriptionsForms/submit.js

```diff
@@ -119,9 +119,11 @@ function buildRegularPaymentRequest(
     deliveryInstructions,
   } = state.page.checkout;
 
+  const addresses = getAddresses(state);
+
   const price = getProductPrice(
     productPrices,
-    state.page.billingAddress.fields.country,
```

Comment:
This value should not be used directly because it may be wrong if `billingAddressIsSame` is true

Reply:
for long term safety, would it be worth making the "update address" action update both in the state?

Reply:
I was thinking about that, it's quite tricky to do because each address is managed by a separate reducer. It probably is worth investigating though.

Reply:
I was thinking about that, it's quite tricky to do because each address is managed by a separate reducer. It probably is worth investigating though.

Reply:
yes definitely doesn't have to be in this pr, more of a general consideration in case people use auto complete in future